### PR TITLE
docs: remove default form message

### PR DESCRIPTION
**Pull request by InnaAtanasova**
Removed "default" messages from the documentation. There are only 4 types of messages: success, warning, error and information. Also updated the tests for form messages

### CODE REVIEW

File: test/resources/theme-ugly.css [hunk 1]
```diff
@@ -95,4 +95,5 @@
   --sapField_TextColor: orange;
   --sapField_InformationBackground: #bc3f8c;
   --sapField_ReadOnly_Background: #4f9344;
+  --sapTextColor: white;
```
Comment (JKMarkowski) [inline test/resources/theme-ugly.css]: Is it necessary?

Comment (InnaAtanasova) [reply test/resources/theme-ugly.css]: This is theme-ugly. It's for the test playground

File: docs/pages/components/form.md [hunk 2]
```diff
@@ -94,16 +94,14 @@ Do not use the input field if:
     <div class="fd-form-item fd-form-item--horizontal fd-has-margin-none fd-popover">
         <label class="fd-form-label" for="input-1">Default input:</label>
         <div class="fd-form-input-message-group">
-            <input class="fd-input fd-popover__control" type="text" id="input-1" placeholder="Field placeholder text" aria-label="Image label" aria-controls="popoverD1" aria-expanded="false" aria-haspopup="true">
-            <span class="fd-popover__body fd-popover__body--no-arrow fd-form-message" aria-hidden="true" id="popoverD1">Normal message</span>
```
Comment (stefanoScalzo) [inline docs/pages/components/form.md]: If you are removing the form message it should not be wrapped in a fd-form-message-group

File: docs/pages/components/form.md [hunk 3]
```diff
@@ -94,16 +94,14 @@ Do not use the input field if:
...
+            <input class="fd-input" type="text" id="input-1" placeholder="Field placeholder text" aria-label="Image label">
         </div>
     </div>
     <br><br>
     <div class="fd-form-item fd-form-item--horizontal fd-has-margin-none fd-popover">
         <label class="fd-form-label" for="input-1">Compact input:</label>
         <div class="fd-form-input-message-group">
-            <input class="fd-input fd-input--compact fd-popover__control" type="text" id="input-1" placeholder="Field placeholder text" aria-label="Image label" aria-controls="popoverD2" aria-expanded="false" aria-haspopup="true">
-            <span class="fd-popover__body fd-popover__body--no-arrow fd-form-message fd-form-message--compact" aria-hidden="true" id="popoverD2">Normal message</span>
+            <input class="fd-input fd-input--compact" type="text" id="input-1" placeholder="Field placeholder text" aria-label="Image label">
```
Comment (stefanoScalzo) [inline docs/pages/components/form.md]: same here

File: docs/pages/components/form.md [hunk 4]
```diff
@@ -169,8 +167,7 @@ Along with Invalid and Warning, error messages should be displayed below the fie
 <div class="fd-form-item fd-has-margin-none fd-popover">
     <label class="fd-form-label" for="input-1">Normal input:</label>
     <div class="fd-form-input-message-group">
-        <input class="fd-input fd-popover__control" type="text" id="input-1" placeholder="Field placeholder text" aria-label="Image label" aria-controls="popoverB1" aria-expanded="false" aria-haspopup="true">
-        <span class="fd-popover__body fd-popover__body--no-arrow fd-form-message" aria-hidden="true" id="popoverB1">Normal message</span>
+        <input class="fd-input" type="text" id="input-1" placeholder="Field placeholder text" aria-label="Image label">
```
Comment (stefanoScalzo) [inline docs/pages/components/form.md]: same here

File: test/templates/form-message/index.njk [hunk 5]
```diff
@@ -4,27 +4,27 @@
 
 {% block content %}
 
-<h3>Default</h3>
+<h3>Success</h3>
 {% set example %}
-    <span class="fd-popover__body fd-popover__body--no-arrow fd-form-message" aria-hidden="true" id="popoverD1">Normal message</span>
```
Comment (stefanoScalzo) [inline test/templates/form-message/index.njk]: why are you removing these classes?

Comment (InnaAtanasova) [reply test/templates/form-message/index.njk]: because you don't need them. This is not self-contained

**Pull request merged**